internal/handler/http_test.go

```diff
@@ -1,6 +1,6 @@
 // // SPDX-FileCopyrightText: 2021 Comcast Cable Communications Management, LLC
 // // SPDX-License-Identifier: Apache-2.0
-package main
+package handler
```

Comment:
I recommend to rename any test files packages to `{package_name}_test`, e.g.:
```suggestion
package handler_test
```